spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/AbstractRdbmsKeyValueRepository.java

```diff
@@ -65,7 +65,7 @@
 
 	private String findAllQuery;
 
-	private String findAllWhereClauseByKey;
+	protected String findAllWhereClauseByKey;
```

Comment:
this should be able to be private again (could just revert the changes to this class altogether, since the others are whitespace changes)